books/free-programming-books-langs.md

```diff
@@ -2005,6 +2005,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### QML
 
 * [Qt5 Cadaques](http://qmlbook.github.io) - Juergen Bocklage-Ryannel and Johan Thelin (HTML, PDF, ePub) (:construction: *in process*)
+* [Qt6 Book](https://www.qt.io/product/qt6/qml-book/preface-preface) - Johan Thelin, Jürgen Bocklage-Ryannel, Cyril Lorquet (HTML) [(PDF)](https://www.qt.io/hubfs/_website/QML%20Book/qt6book-with-frontpage.pdf) (:construction: *in process*)
```

Comment:
@eshellman what is better?
- https://www.qt.io/product/qt6/qml-book/preface-preface
- https://www.qt.io/product/qt6/qml-book/

Reply:
I like preface, but the other is OK too. But just one link per listing, please.

Reply:
> I like preface, but the other is OK too. But just one link per listing, please.

Do you mean I should remove the PDF link?

Reply:
yes please

Reply:
Removing PDF link...

```suggestion
* [Qt6 Book](https://www.qt.io/product/qt6/qml-book/preface-preface) - Johan Thelin, Jürgen Bocklage-Ryannel, Cyril Lorquet (HTML) (:construction: *in process*)
```

Reply:
Sorry for taking so long. Done